### PR TITLE
Poc/design system

### PR DESCRIPTION
**Pull request by benbesuijen**

### CODE REVIEW

File: packages/component/button/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Usage
+Lorem ipsum dolor
```
Comment (coenkoopmans) [inline packages/component/button/USAGE.md]: :(

File: portal/App.js [hunk 2]
```diff
@@ -9,6 +9,10 @@ import { TopBar, TopBarMenu } from './components/TopBar';
 /**
  * Pages
  */
+import { ButtonData } from './../packages/component/button/data.js';
+
+
+
```
Comment (jordybulten) [inline portal/App.js]: Too many empty lines

File: packages/component/button/data.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import style from './STYLE.md';
+import usage from './USAGE.md';
+
+export const ButtonData = {
+  title: 'Button',
+  category: 'Component',
+  description: '<p class="p">Source files: <a href="https://github.com/kpn/kpn-style/blob/master/packages/component/button" class="a">packages/component/button</a></p>',
```
Comment (coenkoopmans) [inline packages/component/button/data.js]: Pls remove dummy data

File: portal/App.js [hunk 4]
```diff
@@ -9,6 +9,10 @@ import { TopBar, TopBarMenu } from './components/TopBar';
 /**
  * Pages
  */
+import { ButtonData } from './../packages/component/button/data.js';
+
+
```
Comment (coenkoopmans) [inline portal/App.js]: Why so many new lines?

File: portal/assets/style/_jumbotron.scss [hunk 5]
```diff
@@ -17,4 +17,11 @@
       color: color(grey, silver);
     }
   }
+
+  + .tab {
```
Comment (jordybulten) [inline portal/assets/style/_jumbotron.scss]: Indentation failed

File: portal/assets/style/_jumbotron.scss [hunk 6]
```diff
@@ -17,4 +17,11 @@
...
+    .tab__menu {
+      padding-left: space(6);
```
Comment (coenkoopmans) [inline portal/assets/style/_jumbotron.scss]: Space 6 seems like a lot?

File: portal/assets/style/_jumbotron.scss [hunk 7]
```diff
@@ -17,4 +17,11 @@
       color: color(grey, silver);
     }
   }
+
+  + .tab {
+    .tab__menu {
```
Comment (coenkoopmans) [inline portal/assets/style/_jumbotron.scss]: Simplify to 1 selector if possible / clean (like here)

File: portal/components/MarkDownLoader.js [hunk 8]
```diff
@@ -11,24 +12,68 @@ class MarkDownLoader extends React.Component {
...
+            <div className="jumbotron">
+              <div className="jumbotron__body">
+                <div className="jumbotron__title">Button</div>
+              </div>
+            </div>
+
+            <nav className="tab">
+              <ul className="tab__menu">
+                {
+                  this.props.data.content.map((content) =>
```
Comment (coenkoopmans) [inline portal/components/MarkDownLoader.js]: You are not returning anything in this map function, right? So you could use a forEach instead

File: portal/components/MarkDownLoader.js [hunk 9]
```diff
@@ -11,24 +12,68 @@ class MarkDownLoader extends React.Component {
...
   render() {
     return (
-      <Content title={this.props.title}>
-        <ReactMarkdown className="kpn-style" renderers={{code: CodeBlock}} source={this.state.markdown} escapeHtml={false} />
+      <Content title={this.props.data?.category || this.props.title}>
+        {this.props.data && Array.isArray(this.props.data.content) &&
+          <>
+            <div className="jumbotron">
+              <div className="jumbotron__body">
+                <div className="jumbotron__title">Button</div>
```
Comment (jordybulten) [inline portal/components/MarkDownLoader.js]: Hardcoded title

File: portal/components/MarkDownLoader.js (hunk 8 above)
Comment (benbesuijen) [reply portal/components/MarkDownLoader.js]: Unfortunately forEach is not working for inline React templates

**Pull request merged**